devicetypes/smartthings/zigbee-motion-sensor-light-light.src/led-cpx-motion-sensor-child.groovy

```diff
@@ -0,0 +1,53 @@
+/**
+ *  Copyright 2017 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ *  ITM CPX Motion sensor child
+ *
+ *  Author: SAMSUMG LED
+ *  Date: 2021-12-08
+ */		
+ 						
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus						
+import physicalgraph.zigbee.zcl.DataType						
+						
+metadata {
+	definition (name: "ITM CPX Motion sensor child", namespace: "SAMSUNG LED", author: "SAMSUNG LED", ocfDeviceType: "x.com.st.d.sensor.motion") {
+		capability "Motion Sensor"
+		capability "Configuration"
+		capability "Refresh"
+		capability "Health Check"
+		capability "Sensor"
+	}
+	
+	tiles(scale: 2) {
+		multiAttributeTile(name: "motion", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.motion", key: "PRIMARY_CONTROL") {
+				attributeState "active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+				attributeState "inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#cccccc"
+			}
+		}
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+			state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+		main(["motion"])
+		details(["motion", "refresh"])
+	}
+}
+
+def configure() {
+	return zigbee.configureReporting(0x406, 0x0000, 0x18, 30, 600, null)
```

Comment:
This won't send any Zigbee messages, as child device isn't directly "tied" with a device. But since the same message is returned from parent's `configure()` method, it can be easily deleted from here...